lesson_08/cpp/cyclic_functions.cpp

```diff
@@ -0,0 +1,69 @@
+#include "../headers/cyclic_functions.hpp"
+
+void count_numbers_by_sign()
+{
+	int numbers_qty{ 0 }, positive_qty{ 0 }, negative_qty{ 0 }, zero_qty{ 0 };
+
+	std::cout << "Enter numbers quantity:\n";
+	std::cin >> numbers_qty;
+
+	while (--numbers_qty >= 0)
+	{
+		int number{ 0 };
+		std::cin >> number;
+		if (number > 0)
+		{
+			++positive_qty;
+		}
+		else if (number < 0)
+		{
+			++negative_qty;
+		}
+		else
+		{
+			++zero_qty;
+		}
+	}
+
+	std::cout << "Positive: " << positive_qty << "; Negative: " << negative_qty << "; Zero: " << zero_qty << std::endl;
+}
+
+long long factorial_cycle(const int count)
+{
+	long long result{ 0 };
+
+	if (count == 0 || count == 1)
+	{
+		result = 1;
```

Comment:
`return` here as well and you won't need `result` tmp variable at all making your code shorter and cleaner

Reply:
Fixed. Can influence on cyclomatic complexity metric.